lib/vanagon/platform/windows.rb

```diff
@@ -163,9 +163,24 @@ def generate_nuget_package(project)
       # @return [Array] list of commands required to build an msi package for the given project from a tarball
       def generate_msi_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
-        cg_name = "ProductComponentGroup"
-        wix_extensions =  "-ext WiXUtilExtension -ext WixUIExtension"
-        candle_flags =  "-dPlatform=#{@architecture} -arch #{@architecture} #{wix_extensions}"
+        wix_extensions = "-ext WiXUtilExtension -ext WixUIExtension"
+        # Heat command documentation at: http://wixtoolset.org/documentation/manual/v3/overview/heat.html
+        #   dir <directory> - Traverse directory to find all sub-files and directories.
+        #   -ke             - Keep Empty directories
+        #   -cg             - Component Group Name
+        #   -gg             - Generate GUIDS now
+        #   -srd            - Suppress root element generation, we want to reference one of the default root elements
+        #                     INSTALLDIR or APPDATADIR in the directorylist.wxs file, not a newly generated one.
+        #   -sreg           - Suppress registry harvesting.
+        #   -dr             - Root DirectoryRef to point all components to
+        #   -var            - Replace "SourceDir" in the @source attributes of all components with a preprocessor variable
+        app_heat_flags = " -dr INSTALLDIR -v -ke -indent 2 -cg AppComponentGroup -gg -srd -t wix/filter.xslt -sreg -var var.AppSourcePath "
```

Comment:
Thats a directive to heat to use that variable as a reference instead of source dir
If you look down, you will see that we actually give it a value for the candle run.